client/src/components/charts/LineChart.jsx

```diff
@@ -370,6 +367,10 @@ export default class LineChart extends Component {
                     stroke={'#1b1a1e'}
                     tickTextFill={'#1b1a1e'}
                     numTicks={xAxisTicks}
+                    tickLabelProps={val => ({
+                      transform: `rotate(45, ${xScale(val)}, 18)`,
```

Comment:
This is causing a new overlap for date types

![image](https://user-images.githubusercontent.com/2589977/41592685-454ce03e-73b5-11e8-8f4e-2759afe705ea.png)

Reply:
try now. this is why I made the original change
<img width="627" alt="screen shot 2018-06-19 at 2 17 20 pm" src="https://user-images.githubusercontent.com/4628721/41594271-84029506-73cb-11e8-8f10-a21c858ff32d.png">

Reply:
For line charts with tick labels that are short numbers, the axis label is now being pushed out of the visible area. Might need to adjust padding or something

Reply:
![image](https://user-images.githubusercontent.com/2589977/41594813-9c48a452-73bc-11e8-88dc-12ce72901c19.png)